os/arch/arm/src/amebasmart/amebasmart_i2s.c

```diff
@@ -1913,23 +1905,49 @@ static void amebasmart_i2s_suspend(uint16_t port)
 	struct amebasmart_i2s_s *priv = g_i2sdevice[port];
 
 	i2s_disable(priv->i2s_object, 1);
-	i2s_tx_enabled = 0;
```

Comment:
i2s_tx_enabled = 0 is required, please revert this changes.

Reply:
https://github.com/Samsung/TizenRT/blob/e2165ed207bca38e40771c331f4d55c77927cbdc/os/arch/arm/src/amebasmart/amebasmart_i2s.c#L1546-L1551

The amebasmart_i2s_suspend must be called after i2s_stop.
Could you please let me know, is it required??

Reply:
amebasmart_i2s_suspend will be called before board entering PG, but what if I2S_RESUME is called, but I2S_SEND is not called, then it can enter PG even without calling i2s_stop to release the lock. Then in this case amebasmart_i2s_suspend will be called even without i2s_stop.. 
I agree that if the flow is right, this flag is not necessary.

Reply:
is it allow i2s_resume without i2s_send?

Reply:
Porting layer (amebasmart_i2s.c) cannot ensure i2s_send will be called when i2s_resume is called. So I was worried that there might be cases where driver layer didn't call I2S_SEND after I2S_RESUME, and it will cause issue.